webserver.go

```diff
@@ -48,10 +49,11 @@ func serveFavicon(w http.ResponseWriter, r *http.Request) {
 func loginHandler(w http.ResponseWriter, r *http.Request) {
 
 	// create context for template
-	ctx := make(map[string]string)
+	ctx := make(map[string]any)
```

Comment:
Why do you use any here? If because of possible error in line  56 then please check and handle it before populating ctx

Reply:
The template returns an HTML object that cannot be inserted into a `map[string]string`